code/modules/reagents/Chemistry-Holder.dm

```diff
@@ -592,12 +592,7 @@ var/global/obj/temp_reagents_holder = new
 	qdel(R)
 
 /datum/reagents/proc/trans_to_turf(var/turf/target, var/amount = 1, var/multiplier = 1, var/copy = 0, var/defer_update = FALSE) // Turfs don't have any reagents (at least, for now). Just touch it.
-	if(!target || !target.simulated)
-		return
-	var/datum/reagents/R = new /datum/reagents(amount * multiplier, global.temp_reagents_holder)
-	. = trans_to_holder(R, amount, multiplier, copy, TRUE, defer_update = defer_update)
-	R.touch_turf(target)
```

Comment:
I don't see this removal documented anywhere. Is `touch_turf` already called elsewhere in the chain?